racket/collects/net/url-string.rkt

```diff
@@ -176,17 +176,20 @@
            "(?:"              ; | / user-at-opt
```

Comment:
[The comment above](https://github.com/racket/racket/pull/1243/files#diff-69b1d68b3a51a4a74ecf364822d21099L165) claims this regexp matches some RFC — shouldn't that comment be updated?